packages/flutter/test/widgets/widget_inspector_structure_error_test.dart

```diff
@@ -2,93 +2,85 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-import 'dart:convert';
-
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import 'widget_inspector_test_utils.dart';
+
 void main() {
   StructureErrorTestWidgetInspectorService.runTests();
 }
 
-typedef InspectorServiceExtensionCallback = FutureOr<Map<String, Object?>> Function(Map<String, String> parameters);
+class StructureErrorTestWidgetInspectorService extends TestWidgetInspectorService {
+  static void runTests() {
+    final StructureErrorTestWidgetInspectorService service = StructureErrorTestWidgetInspectorService();
```

Comment:
this test version of the service is always enabling structured errors as far as I can tell regardless of whether you are on web.